scripts/install_third_party_libs.py

```diff
@@ -69,6 +69,26 @@
     common.OPPIA_TOOLS_DIR, 'pylint-quotes-0.1.8', 'configparser.py')
 
 
+def tweak_yarn_executable():
+    """When yarn is run on Windows, the file yarn will be executed by default.
+    However, this file is a bash script, and can't be executed directly on
+    Windows. So, to prevent Windows automatically executing it by default
+    (while preserving the behavior on other systems), we rename it to yarn.sh
+    here.
+    """
+    origin_file = os.path.join(common.YARN_PATH, 'bin', 'yarn')
```

Comment:
Let's call this origin_filepath and updated_filepath / new_filepath. (The filepath is the path to the file, and the file is the file object itself.)

Reply:
Updated